aria_data_loaders/aria_data_utils/aria_streamer.py

```diff
@@ -921,14 +921,30 @@ def main(
                 traj_data=vrs_mps_streamer.xyz_trajectory,
                 block=True,
             )
+
+            # Get the position and orientation of the object of interest in spotWorld frame
             pose_of_interest = next_object_spotWorld_T_cpf
+
+            # Position is obtained from the translation component of the pose
             position = pose_of_interest.translation()
-            logger.debug(f" Going to {next_object=} at {position=}")
+
+            # Find the angle made by CPF's z axis with spotWorld's x axis
+            # as robot should orient to the CPF's z axis
+            cpf_z_axis_in_spotWorld = pose_of_interest.matrix()[:3, 2]
+            projected_cpf_z_axis_in_spotWorld_xy = np.multiply(
+                cpf_z_axis_in_spotWorld, np.array([1.0, 1.0, 0.0])
```

Comment:
Could we add a variable name for `np.array([1.0, 1.0, 0.0])`?

Reply:
Addressed here - 707b6e918d97686e3027aeffb58718d5b54c0799